charts/redpanda/templates/rbac.yaml

```diff
@@ -33,9 +33,31 @@ rules:
     - ""
     resources:
       - nodes
+      - secrets
     verbs:
       - get
       - list
+      - watch
+  - apiGroups:
+      - cluster.redpanda.com
+    resources:
+      - redpandas
+    verbs:
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
Maybe I don't understand the side car deployment model. I see that you disables operator mode with `--operator-mode=false`

https://github.com/redpanda-data/helm-charts/pull/684/files#diff-6f494fcb2c4edfa7256b33ba89ef8e7c83d92c7f0a6bf6bea53341881e224620R327

Could you help me understand why Redpandas Role rules are added here?

Reply:
A requirement that was moved over from the roles since I do need to list out the redpandas. However i believe i removed that requirement for the non-operator mode, so it should be safe to remove now.